README.md

```diff
@@ -29,7 +29,32 @@ npmScopes:
 NOTE: If `publishConfig.registry` is specified in `package.json`,
 you must also specify that registry in `npmRegistries` in `.yarnrc.yml`.
 
-3. Run `yarn` commands.
+3. Optionally, add a `.yarn-plugin-aws-codeartifact.yml` configuration file
+   for specifying an [AWS Profile](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html)
+   (can be placed in any parent directory or the home directory, similar to `.yarnrc.yml`):
+```yaml
+npmRegistryServerConfig:
+  awsProfile: my-profile
+# OR
+npmPublishRegistryConfig:
+  awsProfile: my-profile
+# OR
+npmRegistries:
+  //domain-test-000000000000.d.codeartifact.us-east-1.amazonaws.com/npm/repo-test/:
+    awsProfile: my-profile
+# OR
+npmScopes:
+  my-scope:
+    npmRegistryServerConfig:
+      awsProfile: my-profile
+    # OR
+    npmPublishRegistryConfig:
+      awsProfile: my-profile
+```
+NOTE: An `awsProfile` value (including `''`) will override the `AWS_PROFILE` environment variable;
+otherwise, the `AWS_PROFILE` environment variable will be used.
```

Comment:
I would add: " and if `AWS_PROFILE` doesn't exit, the default aws profile will be used, as defined in AWS configuration files (in MacOS and Linux: `~/.aws/credentials`)"